README.md

```diff
@@ -1,15 +1,103 @@
-# MAX Image Caption Generator Web App
+# Generating captions for images and filtering based on image content
 
-A demo web app for MAX using the Image Caption Generator model
+Every day 2.5 quintillion bytes of data are created, based on an [IBM study](https://public.dhe.ibm.com/common/ssi/ecm/wr/en/wrl12345usen/watson-customer-engagement-watson-marketing-wr-other-papers-and-reports-wrl12345usen-20170719.pdf). A lot of that data is unstructured data,
+such as large texts, audio recordings, and images. In order to do something useful
+with the data, we must first convert it to structured data.
 
-## Before Starting the Web App
+In this Code Pattern we will use one of the models from the [Model Asset Exchange](https://developer.ibm.com/code/exchanges/models/),
+an exchange where developers can find and experiment with open source deep learning
+models. Specifically we will be using the [Image Caption Generator](https://github.com/IBM/MAX-Image-Caption-Generator)
+to create a web application that will caption images and allow the user to filter through
+images based image content. The web application provides an interactive user interface
+backed by a lightweight python server using Tornado. The server takes in images via the
+UI and sends them to a REST end point for the model and displays the generated
+captions on the UI. The model's REST endpoint is set up using the docker image
+provided on MAX. The Web UI displays the generated captions for each image as well
+as an interactive word cloud to filter images based on their caption.
 
-Before starting this web app you must setup the MAX Image Caption Generator REST endpoint by following the README at:
+When the reader has completed this Code Pattern, they will understand how to:
 
-[MAX Image Caption Generator GitHub](https://github.com/IBM/MAX-Image-Caption-Generator)
+* Build a Docker image of the Image Caption Generator MAX Model
+* Deploy a deep learning model with a REST endpoint
+* Generate captions for an image using the MAX Model's REST API
+* Run a web application that using the model's REST API
+
+
+# Steps
+
+## Setting up the MAX Model
+
+Note: The set of instructions in this section are a modified version of the one found on the [Image Caption Generator Project Page](https://github.com/IBM/MAX-Image-Caption-Generator)
+
+### 1. Build the Model
+
+Clone the Image Caption Generator model locally. In a terminal, run the following command:
+
+```
+$ git clone https://github.com/IBM/MAX-Image-Caption-Generator.git
```

Comment:
This was brought up in a meeting today: we should be consistent with using `$` in our command lines. I personally prefer leaving them out for an easy copy-paste for the user.

Reply:
Yeah I agree with this. I'll leave them out from now on.